projects/js-packages/publicize-components/src/components/add-connection-modal/constants.tsx

```diff
@@ -0,0 +1,265 @@
+import { SocialServiceIcon } from '@automattic/jetpack-components';
+import { ExternalLink } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import connectionsButtonFacebook from '../../assets/connections-button-facebook.png';
+import connectionsButtonLinkedin from '../../assets/connections-button-linkedin.png';
+import connectionsButtonTumblr from '../../assets/connections-button-tumblr.png';
+import connectionsFacebook from '../../assets/connections-facebook.png';
+import connectionsInstagramBusiness from '../../assets/connections-instagram-business.png';
+import connectionsLinkedin from '../../assets/connections-linkedin.png';
+import connectionsNextdoor from '../../assets/connections-nextdoor.png';
+import connectionsTumblr from '../../assets/connections-tumblr.png';
+
+const sharingButtonLink = 'https://wordpress.com/support/sharing/';
+
+export interface Connection {
+	title: string;
+	icon: ( props ) => JSX.Element;
+	description: string;
+	name: string;
+	examples?: ( () => JSX.Element )[];
```

Comment:
Global `JSX` namespace is deprecated

```suggestion
	icon: ( props ) => React.ReactNode;
	description: string;
	name: string;
	examples?: ( () => React.ReactNode )[];
```